lib/strategies/basic-topic-naming-strategy.spec.ts

```diff
@@ -0,0 +1,16 @@
+import * as fc from 'fast-check';
+import { BasicTopicNamingStrategy } from './basic-topic-naming-strategy';
+
+describe(BasicTopicNamingStrategy.name, () => {
+    it('returns its argument unmodified', () => {
+        const strat = new BasicTopicNamingStrategy();
+
+        // prettier-ignore
+        fc.assert(fc.property(
```

Comment:
Ah man now I gotta go back and do the other strategies like this!

Reply:
Be careful, it's addictive. 👁️